istioctl/cmd/workload.go

```diff
@@ -333,6 +335,29 @@ func createCertsTokens(kubeClient kube.ExtendedClient, wg *clientv1alpha3.Worklo
 	}
 
 	serviceAccount := wg.Spec.Template.ServiceAccount
+	tokenPath := filepath.Join(dir, "istio-token")
+	jwtPolicy, err := util.DetectSupportedJWTPolicy(kubeClient.RESTConfig())
+	if err != nil {
+		fmt.Printf("Failed to determine JWT policy support: %v", err)
```

Comment:
This is not your fault, but please don't use `fmt.Printf()` in istioctl.  Instead, pass `cmd.OutOrStdout()` as a `io.Writer` and use fmt.Fprintf(writer, ...)`.  Use _proxyconfig.go_ as an example.  The reason is that it makes Go tests easier to write.

Reply:
Sure, I have modified to use cmd.OutOrStdout now